docs/api-reference/effects/glfx-shader-modules.md

```diff
@@ -0,0 +1,342 @@
+# glfx Shader Modules
+
+
+Image effects packaged as reusable shader modules in `@luma.gl/effects` based on the [glfx library](http://evanw.github.io/glfx.js/).
+
+
+## Attribution / License
+
+This is a repackaging of shader code from [Evan Wallace](https://github.com/evanw/glfx.js)'s glfx library.
+
+The code and documentation is included here under MIT license.
+
+## Usage
+
+Import brightnessContrast shader module
+
+```js
+    import {brightnessContrast} from @luma.gl/effects;
+```
```

Comment:
This is unclear. Show how to actually use the shader module for rendering, set options etc.

Reply:
We don't have official API to use these shader modules in luma.gl, will add a deck.gl link here.